OpenRA.Mods.Common/Activities/HarvestResource.cs

```diff
@@ -23,15 +24,24 @@ public class HarvestResource : Activity
 		readonly ResourceClaimLayer claimLayer;
 		readonly ResourceLayer resLayer;
 		readonly BodyOrientation body;
+		readonly IMove move;
+		readonly CPos targetCell;
 
-		public HarvestResource(Actor self)
+		public HarvestResource(Actor self, CPos targetcell)
 		{
 			harv = self.Trait<Harvester>();
 			harvInfo = self.Info.TraitInfo<HarvesterInfo>();
 			facing = self.Trait<IFacing>();
 			body = self.Trait<BodyOrientation>();
+			move = self.Trait<IMove>();
 			claimLayer = self.World.WorldActor.Trait<ResourceClaimLayer>();
 			resLayer = self.World.WorldActor.Trait<ResourceLayer>();
+			this.targetCell = targetcell;
+		}
+
+		protected override void OnFirstRun(Actor self)
+		{
+			claimLayer.TryClaimCell(self, targetCell);
```

Comment:
Can you please add a comment above this line explaining why we are ignoring the return value?

Reply:
Done.